src/PharoLauncher-CLI-Tests/PhLCliCommandTest.class.st

```diff
@@ -120,24 +126,30 @@ PhLCliCommandTest >> testFlagValueFor [
 ]
 
 { #category : #test }
-PhLCliCommandTest >> testImplicitDelimiter [
+PhLCliCommandTest >> testImplicitDelimiterIsColonWhenRowModeActibe [
 
-	"implicit value of not implemented/missing flag with parameter should return nil"
 	| aCommand aMatch|
 	
 	aCommand :=  PhLCliCommand new.
-	aMatch := (context arguments: #('launcher')) doMatch.
-	aCommand setArguments: aMatch.
-	self assert: aCommand implicitDelimiter equals: ' '.
-	
-	"now implement using rowMode, it should be with colon"
 	context specification add: aCommand class rowModeFlag.
 	aMatch := (context arguments: #('launcher' '--rowMode')) doMatch.
 	aCommand setArguments: aMatch.
 	
 	self assert: aCommand implicitDelimiter equals: ': '.
 ]
 
+{ #category : #test }
+PhLCliCommandTest >> testImplicitDelimiterIsSpace [
+
+	"implicit value of not implemented/missing flag with parameter should return nil"
```

Comment:
It seems this comment is some copy&paste, maybe we should remove this, because nil value isn't anyhow tested here.